tests/test_controller.py

```diff
@@ -461,32 +461,27 @@ def test_load_licensepooldelivery(self):
     def test_apply_borrowing_policy_when_holds_prohibited(self):
         with self.request_context_with_library("/"):
             patron = self.controller.authenticated_patron(self.valid_credentials)
-            with temp_config() as config:
```

Comment:
A lot of the changes to test code are indentation changes caused by the removal of a `with temp_config()` block.